src/chi-vue/src/components/data-table-filters/AdvancedFiltersPopoverFooter.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="-d--flex" slot="chi-popover__footer">
+    <button
+      @click="clear"
+      class="chi-button -flat -icon -py--0"
+      aria-label="Clear all filters"
+      :disabled="disabledButtons"
+    >
+      <div class="chi-button__content -flex--column -align-items--center">
```

Comment:
@josecarlosgonzalezv why don't you use `UTILITY_CLASSES` here? Please include it